src/component/Legend.tsx

```diff
@@ -27,13 +27,14 @@ function renderContent(content: ContentType, props: Props) {
   if (React.isValidElement(content)) {
     return React.cloneElement(content, props);
   }
-  if (_.isFunction(content)) {
+  if (isFunction(content)) {
     return React.createElement(content as any, props);
   }
 
-  const { ref, ...otherProps } = props;
+  const propsWithoutRef = { ...props };
+  delete propsWithoutRef.ref;
```

Comment:
The other way exposes a `ref` variable that is unused. This approach does the same thing without creating a dangling reference.